harvester/civic.py

```diff
@@ -76,10 +76,7 @@ def convert(gene_data):
                         'description': drug['name'],
                         'id': drug['pubchem_id']
                     })
-                association['phenotype'] = {
-                    'description': evidence_item['disease']['name'],
-                    'id': evidence_item['disease']['url']
```

Comment:
Ah. Okay. I thought it was redundant since it's being added to everything by disease_normalizer. 

Though I suppose in the case where the disease_normalizer doesn't return any results that it would be beneficial to have it there.